manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: FastCopy.FastCopy
+PackageVersion: 4.0.1
+PackageName: FastCopy
+Publisher: H.Shirouzu
+License: GPL 3.0

```

Comment:
```suggestion
License: GPL-3.0
```

Reply:
Thank you for the suggestion.
I update the file based on the software license page.
https://fastcopy.jp/help/fastcopy_eng.htm#license